app/build.gradle.kts

```diff
@@ -43,6 +41,7 @@ android {
             signingConfig = signingConfigs.getByName("release")
         }
         debug {
+            compileSdkPreview = "UpsideDownCake"
```

Comment:
If it's only going to be for debug builds, I'm hesitant that we are inadvertently building something that would be used before the general public.

What I mean by this is that we would only be verifying the version with this flag set, so if there's an issue with the non-debug version, I'm not sure there are tests that help verify that.

With this in mind, is there an issue with removing this entirely then updating the various `compileSdk`, `targetSdk` and the `minimumSdk` properties?